LINSTORPlugin.pm

```diff
@@ -103,6 +123,9 @@ sub options {
         content       => { optional => 1 },
         disable       => { optional => 1 },
         nodes         => { optional => 1 },
+        apicrt        => { optional => 1 },
+        apikey        => { optional => 1 },
+        apica        => { optional => 1 },
```

Comment:
you might want to add an extra space to keep the alignment with the other options.

Reply:
I somehow missed that. Fixed in commit e1c2916.